package.json

```diff
@@ -39,6 +40,7 @@
   "author": "rt2zz <zack@root-two.com>",
   "license": "MIT",
   "dependencies": {
+    "@types/redux": "^3.6.0",
```

Comment:
I am concerned about the bytes that the redux typings will add. Shouldnt the typings field be sufficient and we can may @types/redux a devDependency for the sake of development?